sdk/main/include/TLSBehavior.h

```diff
@@ -0,0 +1,44 @@
+/*-
+*
+* Hedera C++ SDK
+*
+* Copyright (C) 2020 - 2022 Hedera Hashgraph, LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+#ifndef HEDERA_SDK_CPP_TLSBEHAVIOR_H
+#define HEDERA_SDK_CPP_TLSBEHAVIOR_H
+
+namespace Hedera
+{
+
+/**
+ * Enum representing different modes of network operation
+ */
+enum TLSBehavior
```

Comment:
Are these going to be the only two possible options for `TLSBehavior`? If so, would it make sense just to use a boolean instead?

Reply:
More options are expected in the future, similar to https://vertx.io/docs/apidocs/io/vertx/pgclient/SslMode.html